coderd/provisionerdserver/provisionerdserver.go

```diff
@@ -898,10 +898,11 @@ func (s *server) FailJob(ctx context.Context, failJob *proto.FailedJob) (*proto.
 					s.Logger.Error(ctx, "marshal workspace resource info for failed job", slog.Error(err))
 				}
 
-				audit.BuildAudit(ctx, &audit.BuildAuditParams[database.WorkspaceBuild]{
+				audit.WorkspaceBuildAudit(ctx, &audit.BuildAuditParams[database.WorkspaceBuild]{
 					Audit:            *auditor,
 					Log:              s.Logger,
 					UserID:           job.InitiatorID,
+					OrganizationID:   workspace.OrganizationID,
```

Comment:
Ah, that's the trick 👍 I assumed that OrganizationID should be passed together with UserID, but didn't think about pulling it out from workspace.